tests/integration/src/test/scala/mesosphere/marathon/integration/setup/MesosTest.scala

```diff
@@ -42,13 +42,16 @@ case class MesosCluster(
     waitForMesosTimeout: FiniteDuration = 5.minutes,
     mastersFaultDomains: Seq[Option[FaultDomain]],
     agentsFaultDomains: Seq[Option[FaultDomain]],
-    agentsGpus: Option[Int] = None)(implicit
+    agentsGpus: Option[Int] = None,
+    agentSeccompConfigDir: Option[String],
+    agentSeccompDefaultProfile: Option[String])(implicit
     system: ActorSystem,
     mat: Materializer,
     ctx: ExecutionContext,
     scheduler: Scheduler) extends AutoCloseable with Eventually {
   require(quorumSize > 0 && quorumSize <= numMasters)
   require(agentsFaultDomains.isEmpty || agentsFaultDomains.size == numSlaves)
+  require(agentSeccompConfigDir.isEmpty || (agentSeccompConfigDir.isDefined && config.isolation.get.contains("linux/seccomp")))
```

Comment:
Can this be expressed as if/else? I find myself having to think about these clever usages of boolean logic for a substantial amount of time.

Reply:
bump @zen-dog